manifests/init.pp

```diff
@@ -63,13 +56,13 @@
   }
 
   # Validate OS
-  case $::operatingsystem {
+  case $::lsbdistid {
```

Comment:
I had noticed this and as travis nicely points out, this is what is breaking rspec tests. What I don't understand is 'why' the error occurs. As I mentioned in one of the previous commit messages, `apt` requires some facts like `lsbdistcodename`, `lsbdistrelease`, `lsbdistid`, `puppetversion` for Ubuntu tests (https://github.com/puppetlabs/puppetlabs-apt/blob/master/manifests/params.pp#L11-L13). It does not understand operatingsystem* vars. For this reason, the lsbdist* vars were needed. Now, [lsbdistid](https://docs.puppet.com/facter/latest/core_facts.html#lsbdistid) and [operatingsystem](https://docs.puppet.com/facter/latest/core_facts.html#operatingsystem) essentially return the name of the operating system, [lsbdistrelease](https://docs.puppet.com/facter/latest/core_facts.html#lsbdistrelease) and [operatingsystemrelease](https://docs.puppet.com/facter/latest/core_facts.html#operatingsystemrelease) return the operating system release, ... and so on. So, for the sake of uniformity (and also to avoid duplication of similar facts for Ubuntu), I updated all the operatingsystem* vars to lsbdistvars*. As I already mentioned, I see no errors for Puppet 4.9.0. Maybe, the error is due to the caveat of [lsbdistid](https://docs.puppet.com/facter/latest/core_facts.html#lsbdistid):
```
 Caveats:
    Linux: Requires that the lsb_release utility be installed.
```
If that is not the case, then simply because of different puppet versions?